src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,56 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+
 public class WorkWithFile {
+
     public void getStatistic(String fromFileName, String toFileName) {
+        writeToFile(toFileName, createReport(readCsvFile(fromFileName)));
+    }
 
+    private void writeToFile(String file, String data) {
+        File uptputFile = new File(file);
+        try {
+            Files.write(uptputFile.toPath(), data.getBytes());
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
     }
+
+    private String createReport(String data) {
+        StringBuilder report = new StringBuilder();
+        int supply = 0;
+        int buy = 0;
+        String[] record = data.split(System.lineSeparator());
+        for (String singleLineEntry : record) {
+            String[] recordSplit = singleLineEntry.split(",");
+            if (recordSplit[0].equalsIgnoreCase("buy")) {
```

Comment:
Do not use magic numbers
```suggestion
            if (recordSplit[OPERATION_INDEX].equalsIgnoreCase("buy")) {
```

Reply:
Corrected